src/Parser/Shaman/Restoration/Modules/Spells/HealingRain.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+// dependencies
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+
+class HealingRain extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  HealingRainTicks = [];
+
+  get averageHitsPerTick(){
+    let totalHits = 0;
+    this.HealingRainTicks.forEach(x=>totalHits=totalHits+x.hits);
+    return totalHits/this.HealingRainTicks.length;
+  }
+
+  suggestions(when) {
+    when(this.averageHitsPerTick).isLessThan(5)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Try to always cast <SpellLink id={SPELLS.HEALING_RAIN_CAST.id} /> in areas where players stack. This allows the spell to consitantly hit all 6 possible targets.</span>)
+          .icon(SPELLS.HEALING_RAIN_CAST.icon)
+          .actual(`${this.averageHitsPerTick.toFixed(2)} average targets healed`)
+          .recommended(`5 average targets healed`)
+          .regular(5).major(3);
+      });
+  }
+
+  get suggestionThreshold(){
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.

Reply:
In fact, the thresholds here do not match.